internal/collectors/github/organization_collector.go

```diff
@@ -126,3 +142,25 @@ func (c *organizationCollector) collectOrgSamlData(org string) (*bool, error) {
 	return &samlEnabled, nil
 
 }
+
+func (c *organizationCollector) collectEnterpriseVisibilityChangePolicyDisabled() (*bool, error) {
+	enterpriseName, enterpriseExist := context_utils.GetEnterprise(c.Context)
+	visibilityChangeEnabled := false
+	if enterpriseExist {
+		if len(enterpriseName) == 0 {
```

Comment:
1. use strings comparison
2. If enterpriseExists is true i think we can skip the string comparison. 
    a. It's better to test that enterprise is not empty in the context_utils.GetEnterprise() method